libs/docs/src/docs/2.1.0/quickstart-che.md

```diff
@@ -0,0 +1,370 @@
+---
+title: "Quick start with Eclipse Che"
+description: "Quick start with Eclipse Che"
+---
+
+This guide will run through creating a simple hello world devfile project using Eclipse Che. The purpose of this guide is to provide a practical introduction to those just starting out with devfiles.
+
+## Prerequisites
+
+- Eclipse Che 7.62.x
+- `kubectl` or `oc`
+- Access to a Kubernetes or OpenShift cluster
+
+## Procedure
+
+1. Obtain access to Eclipse Che if you do not already have it, as an individual this can be done one of the following ways
+    1. Sign in / Sign up with a free Red Hat account and use [Che Developer Sandbox](https://developers.redhat.com/developer-sandbox/ide)
+    2. Setup a [local instance of Eclipse Che](https://www.eclipse.org/che/docs/stable/administration-guide/installing-che-locally/)
+        - (Optional) You can use `minikube` to run your cluster locally, follow [these](https://minikube.sigs.k8s.io/docs/start/) steps to get started
+
+2. For this quick start guide, we will create a simple [hello world Express.js](https://expressjs.com/en/starter/hello-world.html) application
+
+3. In Eclipse Che, create an empty workspace
+
+4. Close and stop the workspace
+
+5. Go to 'Edit the workspace' and view the devfile. The devfile should be populated with content similar to the following
+    - The `schemaVersion` of the devfile should be left to what Eclipse Che supports, currently `2.1.0`
+    - The `metadata.namespace` field is the target namespace your project will be deployed in, for our current intent and purposes this can be left alone
```

Comment:
This step will not work in Eclipse Che -- https://github.com/eclipse/che/issues/22115. I believe the plan going forward is to allow editing the devfile from within the workspace itself (using the Code editor, for example) but I'm not 100% sure of the status of this feature.

Also, to add, Che currently uses devfile 2.2.0 schema -- the display in the edit page is a bug.

Reply:
> This step will not work in Eclipse Che -- [eclipse/che#22115](https://github.com/eclipse/che/issues/22115). I believe the plan going forward is to allow editing the devfile from within the workspace itself (using the Code editor, for example) but I'm not 100% sure of the status of this feature.
> 
> Also, to add, Che currently uses devfile 2.2.0 schema -- the display in the edit page is a bug.

Thanks for your review @amisevsk!

Seems like the version of Eclipse Che under Red Hat Dev Spaces is an older version that is still on devfile spec `2.1.0`, I will need to write up revised draft for the current `2.1.0` guide as well as a new guide under `2.2.0` to cover the up to date content: https://github.com/devfile/api/issues/1057#issuecomment-1539069527

Reply:
**Update**: https://github.com/devfile/api/issues/1057#issuecomment-1549794771

@amisevsk I need a review of the recent changes when you are able to, thank you.